### ISSUE TITLE
False "should use copy() instead of a loop" positive on copy from slice to map

### ISSUE THREAD

**Issue by hectorj** [dominikh/go-simple#20]
Hi !
Here is an example of code triggering a false-positive:

``` go
mySlice := []string{"testValue"}
myMap := make(map[int]string, len(mySlice))
for k,v := range mySlice {
        myMap[k] = v
}
```

Using a freshly `go get -u`'d version of gosimple, I get the "should use copy() instead of a loop" on this.

But `copy()` cannot be used with a map.

**Comment by hectorj** [comment]
Working on a pull request, coming soon

**Comment by dominikh** [comment]
Closed by 264f25f0dbb049dd9e46440044816cda39ba0343
